templates/data/opensearch/index.html

```diff
@@ -0,0 +1,528 @@
+{% extends 'base_index.html' %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1nVdoKoPrXw8lsvHEtMichlPNfxYU9Uj1zmKowcUXQ_w/edit
+{% endblock meta_copydoc %}
+
+{% block title %}An enterprise  solution for OpenSearch from Canonical{% endblock %}
+
+{% block canonical_url %}{{ request.host_url }}data/opensearch{% endblock %}
+
+{% block meta_description %}
+  Charmed OpenSearch delivers up to 10 years of support and security maintenance for OpenSearch as an integrated, turnkey solution with advanced management features.
+{% endblock %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+  <section class="p-section--hero">
+    <div class="row">
+      <div class="col-3 col-medium-2">
+        <div class="u-hide--small u-vertically-center u-aspect-ratio--16-9"
+             style="margin-top: -55px">
+          {{ image (
```

Comment:
please remove space between `image` and the bracket and reformat, and do the same throughout the whole file